taglib/asf/asfattribute.cpp

```diff
@@ -251,7 +251,7 @@ int ASF::Attribute::dataSize() const
     if(d->pictureValue.isValid()) {
       return d->pictureValue.dataSize();
     }
-    break;
+    return d->byteVectorValue.size();
```

Comment:
why not just [[fallthrough]] ?

Reply:
I thought it was clearer to duplicate the return than fall through since there isn't a logical relationship between GuidType and BytesType other than the internal storage. But I don't have strong feelings about it.

Reply:
It's moot AFAIK. The compiler should be able to optimize it away.